snmp/tests/compose/data/if_counter.snmprec

```diff
@@ -0,0 +1,2364 @@
+1.3.6.1.2.1.1.1.0|4x|4249472d4950205669727475616c2045646974696f6e203a204c696e757820332e31302e302d3836322e31342e342e656c372e76652e7838365f3634203a204249472d495020736f6674776172652072656c656173652031352e302e312c206275696c6420302e302e3131
```

Comment:
Is this new `if_counter` snmprec used ?

Reply:
I use it for generating metrics for staging. I committed because I thought it could be useful, but I don't use it in tests. I will remove it.